dists/win32/ScummVM.iss

```diff
@@ -157,95 +163,94 @@ begin
 end;
 
 [Languages]
-Name: en; MessagesFile: compiler:Default.isl
-Name: br; MessagesFile: compiler:Languages\BrazilianPortuguese.isl
-Name: ca; MessagesFile: compiler:Languages\Catalan.isl
-Name: cz; MessagesFile: compiler:Languages\Czech.isl
-Name: da; MessagesFile: compiler:Languages\Danish.isl
-Name: fr; MessagesFile: compiler:Languages\French.isl
-Name: de; MessagesFile: compiler:Languages\German.isl
-Name: hu; MessagesFile: compiler:Languages\Hungarian.isl
-Name: it; MessagesFile: compiler:Languages\Italian.isl
-Name: nb; MessagesFile: compiler:Languages\Norwegian.isl
-Name: pl; MessagesFile: compiler:Languages\Polish.isl
-Name: ru; MessagesFile: compiler:Languages\Russian.isl
-Name: es; MessagesFile: compiler:Languages\Spanish.isl
-Name: se; MessagesFile: compiler:Languages\Swedish.isl
-
-[Icons]
+Name: english; MessagesFile: compiler:Default.isl
+Name: portuguese; MessagesFile: compiler:Languages\BrazilianPortuguese.isl
+Name: catalan; MessagesFile: compiler:Languages\Catalan.isl
+Name: czech; MessagesFile: compiler:Languages\Czech.isl
+Name: danish; MessagesFile: compiler:Languages\Danish.isl
+Name: french; MessagesFile: compiler:Languages\French.isl
+Name: german; MessagesFile: compiler:Languages\German.isl
+Name: hungarian; MessagesFile: compiler:Languages\Hungarian.isl
+Name: italian; MessagesFile: compiler:Languages\Italian.isl
+Name: norwegian; MessagesFile: compiler:Languages\Norwegian.isl
+Name: polish; MessagesFile: compiler:Languages\Polish.isl
+Name: russian; MessagesFile: compiler:Languages\Russian.isl
+Name: spanish; MessagesFile: compiler:Languages\Spanish.isl
+Name: swedish; MessagesFile: compiler:Languages\Swedish.isl
+Name: basque; MessagesFile: compiler:Languages\Basque.isl
+Name: belarusian; MessagesFile: compiler:Languages\Belarusian.isl
+Name: dutch; MessagesFile: compiler:Languages\Dutch.isl
+Name: finnish; MessagesFile: compiler:Languages\Finnish.isl
+Name: galician; MessagesFile: compiler:Languages\Galician.isl
+Name: greek; MessagesFile: compiler:Languages\Greek.isl
+Name: polish; MessagesFile: compiler:Languages\Polish.isl
+Name: ukrainian; MessagesFile: compiler:Languages\Ukrainian.isl
+
+[Tasks]
+Name: "desktopicon"; Description: "{cm:CreateDesktopIcon}"; GroupDescription: "{cm:AdditionalIcons}" 
+
+[Icons]
+;Optional desktop icon

```

Comment:
Please add a space after the ; for consistency with the existing comments (and the comments in our source code)